docs/src/modules/java/nav.adoc

```diff
@@ -11,5 +11,6 @@
 *** xref:java:timers.adoc[Timers]
 *** xref:java:component-and-service-calls.adoc[Component and Service Calls]
 *** xref:java:failures-and-errors.adoc[Failures and Errors]
+*** xref:java:serialization.adoc[Serialization]
```

Comment:
I think "Serialization" is too specific for this top-level sdk section... What do you think about "Schema Evolution", "API evolution", "Data something?"? Maybe some other you can think of? just trying to brainstorm a bit..

Reply:
It's funny because on a second thought I think "Serialization" is both more generic and more specific than "Schema Evolution" (or similar).. I guess it depends on how you read it. Still think we can probably find a better title .. For now, its mostly about "Schema Evolution", but I could also see this section include more things on how to evolve your application..

Reply:
I'm fine with any name, let's wait for others' input :)

Reply:
OK, let's keep `serialization` for now, and `schema evolution` is one of the serialization aspects, but not the only one.